WalletWasabi/Wallets/Wallet.cs

```diff
@@ -70,8 +70,7 @@ private set
 			StateChanged?.Invoke(this, _state);
 		}
 	}
-
```

Comment:
You can remove the StartupTime property as well.

Reply:
removed.